etckeeper: update to 1.18.14.

mobinmob opened the pull request.

Johnnynator commented:
Can't the python plugins use py3? At least Arch is packaging etckeeper with py3 instead of 2

mobinmob commented:
They probably can... I will test them asap.

13 Μαρ 2020 16:13:26 John Zimmermann <notifications@github.com>:

> 
> 
> 
> Can't the python plugins use py3? At least Arch is packaging etckeeper with py3 instead of 2
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub [https://github.com/void-linux/void-packages/pull/20062#issuecomment-598739606] , or unsubscribe [https://github.com/notifications/unsubscribe-auth/AJOPL3FJSARWRKP337HNNNTRHI5PDANCNFSM4LHANPDA] . [https://github.com/notifications/beacon/AJOPL3A3K7AU5QLUZYQF5PLRHI5PDA5CNFSM4LHANPDKYY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2ZLOORPWSZGOEOYAVFQ.gif]
>

mobinmob commented:
From the package manager plugins only zypper has a hasbang.
Archlinux package is patched to use [breezy](https://pypi.org/project/breezy/) instead of the python2-only bzrtools in order to interface with bazaar repos.
The python dep is needed for bzr support and dnf support.
We have neither dnf nor zypper in the repos. 
I removed the python from hostmakedepends and the bazaar reference from the short_desc and added README.mdwn as documentation with vdoc.
